<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoxColliderWidget</class>
 <widget class="QWidget" name="BoxColliderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>83</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>83</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>63</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="BoxLayout">
    <item row="0" column="2">
     <widget class="QLabel" name="YLabel">
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="XLabel">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="MinValueLabel">
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QDoubleSpinBox" name="SizeZ"/>
    </item>
    <item row="1" column="2">
     <widget class="QDoubleSpinBox" name="SizeY"/>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="SizeX"/>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="ZLabel">
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
